infra/base-images/base-builder/compile_go_fuzzer

```diff
@@ -48,7 +48,14 @@ if [[ $SANITIZER = *coverage* ]]; then
   abspath_repo=`go list -m $tags -f {{.Dir}} $fuzzed_repo || go list $tags -f {{.Dir}} $fuzzed_repo`
   # give equivalence to absolute paths in another file, as go test -cover uses golangish pkg.Dir
   echo "s=$fuzzed_repo"="$abspath_repo"= > $OUT/$fuzzer.gocovpath
-  go test -run Test${function}Corpus -v $tags -coverpkg $fuzzed_repo/... -c -o $OUT/$fuzzer $path
+  # additional packages for which to get coverage
+  pkgaddcov=""
+  if [[ -n "${GO_COV_ADD_PKG}" ]]; then
+    pkgaddcov=","$GO_COV_ADD_PKG
+    abspath_repo=`go list -m $tags -f {{.Dir}} $GO_COV_ADD_PKG || go list $tags -f {{.Dir}} $GO_COV_ADD_PKG`
+    echo "s=$GO_COV_ADD_PKG"="$abspath_repo"= >> $OUT/$fuzzer.gocovpath
+  fi
+  go test -run Test${function}Corpus -v $tags -coverpkg $fuzzed_repo/...$pkgaddcov -c -o $OUT/$fuzzer $path
```

Comment:
What is this change?

Reply:
This change is meant to have better Golang coverage reports.
Now, Golang coverage reports only care about one repository
With this change, you can specify an additional package/repository to get coverage for.
This is useful for ngolo-fuzzing where the fuzz targets live in another repository than the code they are fuzzing

I can split the PR in 2 if you prefer